heartbeat/asterisk

```diff
@@ -308,7 +308,7 @@ asterisk_monitor() {
     #   start.
    
     #To avoid the case where the sipsak check runs before the sip starts at the start action
-    SIPCHECK=`sipsak -s $OCF_RESKEY_monitor_sipuri`
+    SIPCHECK=`sipsak -s '$OCF_RESKEY_monitor_sipuri'`
```

Comment:
I just noticed you should use double-quotes and not backwards quotes around the command here. \` quotes makes SIPCHECK contain the output of the command, which ocf_run will then try to run later.

Reply:
True, just tested it, the correct way will to put it to double-quotes (") and delete a ' from inside from variable
 SIPCHECK="sipsak -s $OCF_RESKEY_monitor_sipuri"

Reply:
Looks good now.

Can you squash the commits? Use `git rebase -i HEAD~2` and replace "pick" with "s" on the last line.

Reply:
> 

sure.
when I wanna rebase, there are 2 commit, 1st my sip uri fix the second one which I fetched from upstream db2...
Should I drop the second one and just leave with squash my one ?

Reply:
Just squash the 2, and then remove the 2nd line from the new commit message.

Reply:
> 

ok, like this. can you check please?

Reply:
You also got the db2 patch that's already been merged in there.

If you do `git checkout HEAD^ heartbeat/db2` and then run `git commit --amend` you can then run `git rebase upstream/master` to get that patch added before your commit.

Reply:
> 

hmm, its should look like this ? I did a screen shot about git log [https://imgur.com/a/3IticRY](url)

Reply:
It seems like it's missing the merge commit: commit af7a311021116b8b6d317af4c795745d6c115e38 (HEAD -> master, upstream/master)